OrbitGl/TimerTrack.h

```diff
@@ -66,6 +66,8 @@ class TimerTrack : public Track {
   virtual void UpdateBoxHeight();
   [[nodiscard]] virtual float GetYFromDepth(uint32_t depth) const;
 
+  virtual const float GetHeaderHeight() const;
```

Comment:
Please add [[nodiscard]].  Also, returning "const float" is not useful, "float" is perfectly fine.

Reply:
Thanks! I will update the code.